ui/app/common/patient-search/models/search.js

```diff
@@ -95,6 +95,17 @@ Bahmni.Common.PatientSearch.Search = function (searchTypes) {
             return patient[column];
         }).join(" - ");
 
+        var extraIdentifier = null;
+        if (patient.extraIdentifiers) {
+            var objIdentifiers = JSON.parse(patient.extraIdentifiers);
+            for (var key in objIdentifiers) {
+                extraIdentifier = objIdentifiers[key];
+                break;
+            }
+        } else if (patient.extraIdentifierVal) {
```

Comment:
Hi @shilpa-iplit , can you explain what this logic means? I didn't understand.

Reply:
On Line# 99 In case of Clinical Search 'All' tab - search is returning patient.extraIdentifiers with all extra identifiers from bahmni-core code. So here we are picking up the first extra identifier.
On line# 105 For active patients list we are executing query from GP's  "emrapi.sqlSearch.activePatients" which is returning first extra identifier value in model patient as patient.extraIdentifierVal.
GP "bahmni.extraPatientIdentifierTypes" may have list of extra identifiers Here also we are picking up the first extra identifier.
Please advise if all logic is correct.

Reply:
Will request @binduak or @swatigogia2020 to recheck. I think its right.